workflow/metrics/controller_metrics.go

```diff
@@ -2,44 +2,20 @@ package metrics
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/tools/cache"
 
 	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
-	"github.com/argoproj/argo/workflow/util"
 )
 
 // legacyWorkflowCollector collects metrics about all workflows in the cluster
 type controllerCollector struct {
```

Comment:
this file's name does not make the type contained

Reply:
Not sure what you mean here?